README.md

```diff
@@ -217,23 +217,34 @@ kubectl apply -f https://raw.githubusercontent.com/<org>/cluster-api-agent/<tag
 
 E2E tests can be executed through ansible tasks in a target remote host.
 
-The following env vars are needed:
-`SSH_KEY_FILE` key file to access the remote host
-`REMOTE_HOST` remote host where to execute the tests
-`PULLSECRET` base64 encoded pullsecret to inject into the tests
+#### Prerequisites
 
-Then we can run:
+Export the following env vars:
+
+`SSH_KEY_FILE` path to the private SSH key file to access the remote host
+`SSH_AUTHORIZED_KEY` value of your public SSH key
```

Comment:
Used to access cluster nodes, should we specify it in the readme?

Reply:
Yeah good idea!